mozregression/bisector.py

```diff
@@ -276,6 +228,66 @@ def user_exit(self, mid):
         self._logger.info('%s known bad inbound revision: %s'
                           % (words[1], self.bad_revision))
 
+    def handle_merge(self):
+        # let's check if we are facing a merge, and in that case,
+        # continue the bisection from the merged branch.
+        result = None
+
+        self._logger.debug("Starting merge handling...")
+        # so, we have to check the commit of the interesting push
+        # that is the most recent push
+        interesting_push = self.build_range[1]
```

Comment:
I really don't get what "interesting" means here. It just seems to be the end of the build range? Maybe call it `most_recent_push`?